src/NeoSharp.Core/Network/EndPoint.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NeoSharp.Core.Network
+{
+    public class EndPoint
+    {
+        public Protocol Protocol { get; set; }
+
+        public string Host { get; set; }
```

Comment:
i prefer IpAddress or IpEndPoint here

Reply:
Host property can hold name or IP address. I've checked the previous implementation and found WebSocket does not use either IPEndPoint or IPAddress (https://github.com/neo-project/neo/blob/11d8db11568d9eadeeb86c5b8c21a1d3937e0912/neo/Network/LocalNode.cs#L632). I will reconsider this choice when I implement TCP communication and DNS IP resolution. Thanks for the suggestion.